.babelrc

```diff
@@ -1,12 +1,27 @@
 {
-  "presets": ["next/babel"],
-  "plugins": [
-    [
-      "import",
-      {
-        "libraryName": "antd",
-        "style": "css"
-      }
-    ]
-  ]
+    "presets": ["next/babel"],
+    "env": {
```

Comment:
Need to do this because 'import' plugin bug in test env ie https://laysent.github.io/til/2019-07-29_jest-with-ant-design